### DISCUSSION TITLE
Shot Timer not showing live data

### DISCUSSION THREAD

**Discussion by Sagakei** [zweckj/lamarzocco#45] [Q&A]
I am having an issue with the shot timer. All the other entities of the integration work from my HA dashboard, but when I am pulling a shot, the timer isn't updating. I am newer to HA, is there something I am doing wrong?
<img width="656" alt="Screenshot 2024-12-28 at 1 32 32 PM" src="https://github.com/user-attachments/assets/ad1aed1a-4eae-48d0-8272-e17d02b61efb" />

**Comment by zweckj** [comment]
- is this actually the integration from this repo? If yes, switch to the core (HA integrated) one
- if no, reload your integration shortly before pulling a shot and see if that makes a difference

**Reply by Sagakei** [reply]
Thanks for the quick reply. I used HACS previously and remove that integration. When installing from integrated HA list it still lists it as a "Custom Integration". I'll just try a fresh instance of HA, without HACS, and see. 

<img width="384" alt="Screenshot 2024-12-28 at 2 13 43 PM" src="https://github.com/user-attachments/assets/8851ae13-e317-41b9-b1f9-0506fe6c6a83" />

**Reply by zweckj** [reply]
You still have the custom component in the folder apparently
